splinepy/_version.py

```
@@ -1 +1 @@
-__version__ = "0.1.3"
+__version__ = "0.2.0"
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)